kore/src/Kore/Internal/Predicate.hs

```diff
@@ -914,6 +916,9 @@ simplifiedAttribute = Attribute.simplifiedAttribute . extractAttributes
 isSimplified :: SideCondition.Representation -> Predicate variable -> Bool
 isSimplified condition = Attribute.isSimplified condition . extractAttributes
 
+isSimplifiedIgnoreCondition :: Predicate variable -> Bool
```

Comment:
Please add documentation similar to `Kore.Attribute.Pattern.Simplified.isSimplifiedIgnoreCondition`.